src/components/Filters/SchemaSieve.ts

```diff
@@ -289,6 +291,16 @@ export const getPropertyScheme = (
 		: null;
 };
 
+export const getDescriptionProperty = (
```

Comment:
Any reason we need this and can't just use `parseDescription(schemaValue)?.[propertyKey]` on the consumer side?

Reply:
removed, thanks for pointing it out :)